### PR TITLE
Add changed to config files

### LINKED ISSUE #1927: Fix scope naming "config_files" vs. "changed configuration files" in output and documentation
The scope "config_files" is referred to in different naming throughout machinery:
- different output for:
```machinery show <descr>```: title is "Changed configuration files"
 but for ```machinery show --html <descr>```: title is "Config Files"
- The man page even includes the scope "config_files", and talks about "changed configuration files" at the same time
- Other places?

We should only refer to the scope using one and the same wording.

### PR DESCRIPTION
**Pull request by thardeck**
Fixes #1927

### CODE REVIEW

File: plugins/changed_config_files/changed_config_files_inspector.rb [hunk 1]
```diff
@@ -37,25 +37,27 @@ def initialize(system, description)
...
-    do_extract = options[:extract_changed_config_files]
+    do_extract = options[:extract_changed_changed_config_files]
     system.check_retrieve_files_dependencies if do_extract
 
     count = 0
     files = @system.managed_files_database.changed_files do |chunk|
       count += chunk.lines.count { |l| !l.chomp.end_with?(":") && l.split(" ")[1] == "c" }
-      Machinery::Ui.progress(" -> Found #{Machinery.pluralize(count, "%d changed config file")}...")
+      Machinery::Ui.progress(
+        " -> Found #{Machinery.pluralize(count, "%d changed configuration file")}..."
```
Comment (mauromorales) [inline plugins/changed_config_files/changed_config_files_inspector.rb]: Just as a comment I think this would read better if it said "changed configuration file(s)" but we can do that in a different PR otherwise this would be the only scope that reads this way

File: schema/migrations/migrate7to8.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    if File.directory?(File.join(@path, "analyze", "config_file_diffs"))
+      FileUtils.mv(
+        File.join(@path, "analyze", "config_file_diffs"),
+        File.join(@path, "analyze", "changed_config_files_diffs")
+      )
+    end
+
+    if @hash.key?("config_files")
+      @hash["changed_config_files"] = @hash["config_files"]
+      @hash.delete("config_files")
```
Comment (mauromorales) [inline schema/migrations/migrate7to8.rb]: This could be shortened to

```
@hash["changed_config_files"] = @hash.delete("config_files")
```

File: schema/migrations/migrate7to8.rb [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    end
+
+    meta = @hash["meta"]
+    if meta.key?("config_files")
+      meta["changed_config_files"] = meta["config_files"]
+      meta.delete("config_files")
```
Comment (mauromorales) [inline schema/migrations/migrate7to8.rb]: same here

File: schema/migrations/migrate7to8.rb (hunk 2 above)
Comment (thardeck) [inline schema/migrations/migrate7to8.rb]: I know but I have asked andred because I did not find the short form that readable and he also did like the current one more.

File: plugins/changed_config_files/changed_config_files_inspector.rb (hunk 1 above)
Comment (mauromorales) [inline plugins/changed_config_files/changed_config_files_inspector.rb]: ignore previous comment

**Comment by mauromorales** [comment]
lgtm
